### PR TITLE
Use shared URDF for both simulation and Pinocchio

### PR DESCRIPTION
**Pull request by Kotochleb**

### CODE REVIEW

File: urdf/franka_arm.xacro [hunk 1]
```diff
@@ -6,7 +6,7 @@
   <!-- safety_distance: Minimum safety distance in [m] by which the collision volumes are expanded and which is enforced during robot motions -->
   <!-- arm_id: Namespace of the robot arm. Serves to differentiate between arms in case of multiple instances. -->
   <!-- joint_limits: description of the joint limits that comes from a YAML file. Example definition: ${xacro.load_yaml('$(find franka_description)/robots/fr3/joint_limits.yaml')} -->
-  <xacro:macro name="franka_arm" params="arm_id:='panda' description_pkg:='franka_description' connected_to:='' xyz:='0 0 0' rpy:='0 0 0' gazebo:=false safety_distance:=0 joint_limits" >
+  <xacro:macro name="franka_arm" params="arm_id:='panda' description_pkg:='franka_description' connected_to:='' xyz:='0 0 0' rpy:='0 0 0' gazebo:=false pinocchio_colisions:=false safety_distance:=0 joint_limits" >
```
Comment (MaximilienNaveau) [inline urdf/franka_arm.xacro:9]: shouldn't it be `collision` with 2 "l"?

**Review by MaximilienNaveau — APPROVED** [review]
In principle this sounds good.
Though I believe that a independent package from the panda_torque_mpc should be focused on providing the different robot models one needs for control.

The idea is to provide the same kinematics a dynamics but potentially have the collision models in xacro condition in order to load convex collision geometry or not for example.

Two methods are available:
1. we fork [franka_ros](https://github.com/frankaemika/franka_ros/tree/noetic-devel) in the agimus-project organization and provide a PR upstream in order to make things clean.
2. We create a franka_description based on the upstream [franka_description](https://github.com/frankaemika/franka_ros/tree/develop/franka_description) from [franka_ros](https://github.com/frankaemika/franka_ros/tree/noetic-devel). And we use the ROS workspace overlay to use our fork.

I believe the first process is the cleanest.
Moreover I would be in favor of creating a PR for ros2 in the frank_ros package as well if we still intend to pursue the migration of the panda robot we do have.

File: urdf/franka_arm.xacro (hunk 1 above)
Comment (Kotochleb) [reply urdf/franka_arm.xacro:9]: Should be

**Review by MaximilienNaveau — APPROVED** [review]

**Comment by MaximilienNaveau** [comment]
@MedericFourmy can you review/merge this?

**Comment by MedericFourmy** [comment]
Tested in sim and on panda, fixed compilation bug and added rviz config to show simplified collision shapes.

**Review by MaximilienNaveau — APPROVED** [review]

**Comment by MaximilienNaveau** [comment]
@Kotochleb and @MedericFourmy ready to be merged. I believe @MedericFourmy is still the only one who can merge.
It would be nice if @MedericFourmy you could give the permission to @nim65s @Kotochleb and me for merging these PRs.

**Pull request merged**